### PR TITLE
Proofread Watson

### PR DESCRIPTION
**Pull request by B1ggDave**
proofread but there are alot of 
TO DO
line 321
line 199
line 39

Also technical phrase on line 203 bit array turned into bit array?

### CODE REVIEW

File: pages/docs/tutorials/multiplatform-library.md [hunk 1]
```diff
@@ -9,40 +9,40 @@ showAuthorInfo: false
...
 
-We will be using IntelliJ Community Edition for this tutorial, though using Ultimate edition is possible as well. Kotlin plugin 1.3.x or higher should be installed in the IDE. 
-This can be verified via *Language & Frameworks | Kotlin Updates* section in the *Settings* (or *Preferences*) of the IDE.
+We will be using IntelliJ IDEA Community Edition for this tutorial, though using Ultimate edition is possible as well. The Kotlin plugin 1.3.x or higher should be installed in the IDE. 
+This can be verified via the *Language & Frameworks | Kotlin Updates* section in the *Settings* (or *Preferences*) of the IDE.
 Native part of this project is written using Mac OS X, but don't worry if you are using another platform, the platform affects only directory names in this particular tutorial.
 
 # Creating a project
 
 (TODO links on Sergei article)
```
Comment (B1ggDave) [inline pages/docs/tutorials/multiplatform-library.md:35]: TODO

File: pages/docs/tutorials/multiplatform-library.md [hunk 2]
```diff
@@ -9,40 +9,40 @@ showAuthorInfo: false
...
-We will be using IntelliJ Community Edition for the example. You need to make sure you have the latest version of Kotlin plugin installed, 1.3.x or newer.
-We select *File | New | Project*, select *Kotlin | Kotlin (Multiplatform Library)* and configure the project in the way you want. 
+We will be using IntelliJ IDEA Community Edition for the example. You need to make sure you have the latest version of the Kotlin plugin installed, 1.3.x or newer.
+We select *File | New | Project*, select *Kotlin | Kotlin (Multiplatform Library)* and configure the project in the way we want. 
 The only limitation is to use Gradle 4.10 for Kotlin/Native plugin. Let's patch the `gradle/wrapper/gradle-wrapper.properties` and specify gradle version 4.10. 
 TODO Mikhail should fix bugs with wrapper and this part can be removed
```
Comment (B1ggDave) [inline pages/docs/tutorials/multiplatform-library.md:39]: TO DO

File: pages/docs/tutorials/multiplatform-library.md [hunk 3]
```diff
@@ -306,16 +306,16 @@ version '1.0.0'
...
-- have written common tests which are executed on every platform
-- published the final library to Maven repository
+In this tutorial we have:
+- Created a multiplatform library with platform-specific implementations.
+- Provided default implementation for common part and specialized it on JVM.
+- Written common tests which are executed on every platform.
+- Published the final library to Maven repository.
 
-We may find the whole sources from that tutorial on [GitHub] TODO where should sources be?
+We may find all the sources from this tutorial on [GitHub] TODO where should sources be?
```
Comment (B1ggDave) [inline pages/docs/tutorials/multiplatform-library.md:321]: TO DO

File: pages/docs/tutorials/multiplatform-library.md [hunk 4]
```diff
@@ -190,18 +190,18 @@ object NativeBase64Encoder : Base64Encoder {
...
+To make the library complete we should write some tests, but we have three independent implementations and it is a waste of time to write duplicate tests for each one.
+The good thing about common code is that it can be covered with common tests, which later are compiled and executed on *every* platform.
+All the bits for testing are already generated by the project Wizard. TODO link to Sergei article to avoid explanation of `test` source sets and dependencies.
 
-Let's create class `Base64Test` in `commonTest/kotlin/jetbrains/base64` folder and write a basic tests for Base64.
+Let's create the class `Base64Test` in `commonTest/kotlin/jetbrains/base64` folder and write the basic tests for Base64.
 
-But as you remember, our API converts byte arrays to byte arrays and it's not quite handy to test byte arrays.
-So before we start writing a test, let's add a method `encodeToString` with a default implementation to our `Base64Encoder` interface:
+But as you remember, our API converts byte arrays to byte arrays and it's not useful to test byte arrays.
```
Comment (B1ggDave) [inline pages/docs/tutorials/multiplatform-library.md:203]: converts bit arrays to bit arrays?

Comment (qwwdfsad) [reply pages/docs/tutorials/multiplatform-library.md:203]: Yes. Will rephrase as "But as you remember, our API converts byte arrays to byte arrays in a different format" if you don't mind.

Also, why is it "not useful"? It is useful, but tests are not human-readable and that's what I mean by "not quite handy".

Comment (B1ggDave) [reply pages/docs/tutorials/multiplatform-library.md:203]: thanks! Ahh good ok so the original was "not handy" I misinterpreted this whole sentence. 
then let's use "it is not easy to test byte arrays"?  what do you think? Handy == useful . like saying "that hammer is very handy"

**Pull request merged**